site/src/components/keyboard-controls/KeyboardControls.module.css

```diff
@@ -3,14 +3,23 @@
   margin-bottom: calc(2 * var(--salt-size-unit));
 }
 
-.table td {
-  vertical-align: top;
-}
-
 .table td > :last-child {
   margin-bottom: 0;
 }
 
 .keyCol {
   min-width: 10rem;
 }
+
+.key {
+  display: inline-block;
+  background-color: #ffffff;
+  border: var(--salt-size-border) var(--salt-container-borderStyle) var(--salt-container-primary-borderColor);
+  border-radius: 3px;
+  font-family: var(--salt-text-label-fontFamily);
```

Comment:
This might be better off not using label though cc @pseys

Reply:
I caught up with @pseys, his opinion is that we merge this now but we raise an issue to align the styling with Salt later.